tests/target/chains.rs

```diff
@@ -0,0 +1,48 @@
+// Test chain formatting.
+
+fn main() {
+    // Don't put chains on a single list if it wasn't so in source.
+    let a = b.c
+             .d
+             .1
+             .foo(|x| x + 1);
+
+    bbbbbbbbbbbbbbbbbbb.ccccccccccccccccccccccccccccccccccccc.ddddddddddddddddddddddddddd();
+
+    bbbbbbbbbbbbbbbbbbb.ccccccccccccccccccccccccccccccccccccc
+                       .ddddddddddddddddddddddddddd
+                       .eeeeeeee();
+
+    x().y(|| {
+        match cond() {
```

Comment:
This actually tests the case where we do continue after a function call when the length of the function call < tab_spaces.